src/utils/login.test.ts

```diff
@@ -16,6 +16,22 @@ jest.mock('./api', () => ({
   request: require('../../jest/unit/components/__mocks__/api').default.request,
 }));
 
+jest.mock('i18next', () => {
+  const translationEN = require('../../i18n/translations/en-US.json');
```

Comment:
Do we need this? I don't see the connection with the test.

Reply:
in fact they have a connection ... I'm using the English translation file to do the tests ... I didn't want to write consts or a string ... I think it's better if we reuse it ... so let's say I don't I want more "Login", but "Sign in" .. if the id's remained the same (en translations) and only the value changes, we don't need to update the tests